fidesctl/src/fideslang/validation.py

```diff
@@ -14,16 +14,16 @@ class FidesValidationError(Exception):
 
 class FidesKey(ConstrainedStr):
     """
-    A FidesKey should only contain alphanumeric characters, '.' or '_'
+    A FidesKey type that creates a custom constrained string.
     """
 
-    regex: Pattern[str] = re.compile(r"^[a-zA-Z0-9_.]+$")
+    regex: Pattern[str] = re.compile(r"^[a-zA-Z0-9_.-]+$")
```

Comment:
Does this work in Python regexps?
```suggestion
    regex: Pattern[str] = re.compile(r"^[A-z0-9_.-]+$")
```

Reply:
yes